docs/content.zh/release-notes/flink-1.19.md

```diff
@@ -0,0 +1,362 @@
+---
+title: "Release Notes - Flink 1.19"
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release notes - Flink 1.19
+
+These release notes discuss important aspects, such as configuration, behavior or dependencies,
+that changed between Flink 1.18 and Flink 1.19. Please read these notes carefully if you are 
+planning to upgrade your Flink version to 1.19.
+
+## Dependency upgrades
+
+#### Drop support for python 3.7
+
+##### [FLINK-33029](https://issues.apache.org/jira/browse/FLINK-33029)
+
+#### Add support for python 3.11
+
+##### [FLINK-33030](https://issues.apache.org/jira/browse/FLINK-33030)
+
+## Build System
```

Comment:
@snuyanzin I've added the java 21 support, PTAL, thanks : )
(also add it to the release announcement pr:
https://github.com/apache/flink-web/pull/721/files#diff-9a4ae63f4269f0b6dee7a38b30564cc94f59cd21f1e5eb6edb13902c1f8bb3fbR234)

Reply:
great, thanks for adding it